packages/terra-action-footer/package.json

```diff
@@ -21,8 +21,8 @@
   },
   "homepage": "https://github.com/cerner/terra-core#readme",
   "peerDependencies": {
-    "react": "^16.4.2",
-    "react-dom": "^16.4.2",
+    "react": "^16.8.4",
+    "react-dom": "^16.8.4",
     "terra-base": "^4.0.0"
```

Comment:
@bjankord I haven't had a chance to look into the ramifications of this since we last talked about it, but would you be interested in getting rid of these terra-base peer-dependencies with this major version bump if we can determine they're not necessary?

Reply:
If its something we decide to do, now would be a good time to do it. Let's get an issue logged to track thoughts on the change of removing base as a peerDep.

Reply:
Logged #2291

Reply:
Removed terra-base peer dependency across terra-core packages: https://github.com/cerner/terra-core/pull/2288/commits/adc84acfdb1919c67d9140e0cd5780ee2d1f97ac

Reply:
Removed baseStyles import in this commit: https://github.com/cerner/terra-core/pull/2288/commits/01e5365ac3356bb3916420ad8b9ae14b3a9bab23